wasmcloud_host/lib/wasmcloud_host/lattice/control_interface.ex

```diff
@@ -2,7 +2,9 @@ defmodule WasmcloudHost.Lattice.ControlInterface do
   @wasmbus_prefix "wasmbus.ctl."
 
   def scale_actor(actor_id, actor_ref, desired_count, host_id) do
-    topic = "#{@wasmbus_prefix}#{HostCore.Host.lattice_prefix()}.cmd.#{host_id}.scale"
+    {_pk, _pid, prefix} = WasmcloudHost.Application.first_host()
+
+    topic = "#{@wasmbus_prefix}#{prefix}.cmd.#{host_id}.scale"
```

Comment:
I think this needs to use `ctl_topic_prefix`, right?

Reply:
no, the old version was `wasmbus_prefix` before, which actually includes the `ctl` (  `@wasmbus_prefix "wasmbus.ctl."`)

Reply:
I'm thinking about hosts which have a custom ctl topic prefix set, e.g. through `WASMCLOUD_CTL_TOPIC_PREFIX`